src/cc/libbpf.c

```diff
@@ -767,9 +767,12 @@ static int bpf_get_retprobe_bit(const char *event_type)
  * creating [k,u]probe with perf_event_open, which makes it easier to clean up
  * the [k,u]probe. This function tries to create pfd with the perf_kprobe PMU.
  */
+
+#define PERF_UPROBE_REF_CTR_OFFSET_SHIFT 32  // from linux/trace_events.h
```

Comment:
This header wasn't available on the machine i tested, so I'm hardcoding it for compatibility. It seems unlikely to change, but perhaps we could try and detect it instead and fallback to hardcoded only if needed?